agent/containers/images/Dockerfile.layered.j2

```diff
@@ -7,4 +7,4 @@ FROM pbench-agent-base-{{ distro }}:{{ tag }}
 #        Kubernetes or RHV environments.
 RUN {% if distro == 'centos-7' %}yum{% else %}dnf{% endif %} install -y {% if distro == 'centos-8' %}--enablerepo PowerTools {% endif %}--enablerepo copr-pbench {{ rpms }} && \
     {% if distro == 'centos-7' %}yum{% else %}dnf{% endif %} -y clean all && \
-    rm -rf /var/cache/yum
+    rm -rf /var/cache/{% if distro == 'centos-7' %}yum{% else %}dnf{% endif %}
```

Comment:
I wondered when I first saw this in the original PR why you didn't use `{{ pkgmgr }}` ... I figured it was probably good because this is technically a different context and while the `/var/cache/` directories apparently happen to match, it might be best not to depend on that.

However, I notice that in Dockerfile.base, you _did_ use `{{ pkgmgr }}` in the same context; it'd be good to be consistent, one way or the other.

(And yes, I know this is not the sort of comment one likes to see in a backport, which is of course why I noticed it now and not earlier!)

Reply:
We can't use `pkgmgr` for the other jinja2 invocations because it is not defined.  Only the base images have it defined explicitly.  I need to move the template generation into the image build script so that I don't need that in the `Makefile`.

Reply:
This is the kind of thing I'd like to fix on `master`, but not backport.